core/src/main/java/edu/stanford/futuredata/macrobase/contrib/aria/APICubeResult.java

```diff
@@ -0,0 +1,10 @@
+package edu.stanford.futuredata.macrobase.contrib.aria;
+
+import java.util.List;
+
+public class APICubeResult {
+    public String interval;
+    public String granularity;
```

Comment:
comments here to describe what each field means would be really useful; e.g., what is interval vs granularity?